src/databricks/labs/ucx/installer/policy.py

```diff
@@ -0,0 +1,131 @@
+import json
+import logging
+
+from databricks.labs.blueprint.installation import Installation
+from databricks.labs.blueprint.installer import InstallState
+from databricks.labs.blueprint.tui import Prompts
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import NotFound
+from databricks.sdk.service import compute
+
+logger = logging.getLogger(__name__)
+
+
+class ClusterPolicyInstaller:
+    def __init__(self, installation: Installation, ws: WorkspaceClient, prompts: Prompts):
+        self._ws = ws
+        self._installation = installation
+        self._prompts = prompts
+
+    @staticmethod
+    def _policy_config(value: str):
+        return {"type": "fixed", "value": value}
+
+    def create(self, inventory_database: str) -> tuple[str | None, str | None, dict | None]:
```

Comment:
```suggestion
    def create(self, inventory_database: str) -> tuple[str, str | None, dict]:
```
avoid ` | None` like plague, try to return "zero values" when possible